package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "grommet",
-  "version": "2.17.0",
+  "version": "2.16.3",
```

Comment:
Please clean up your branch from merge conflicts, I'll hold off the review until you'll take a look at it.

Reply:
Thanks. will do.

Reply:
All merge conflicts have been resolved @ShimiSun